crates/matrix-sdk/src/client/mod.rs

```diff
@@ -358,6 +371,8 @@ impl Client {
     /// * [`Room`][room::Room] is only available for room-specific events, i.e.
     ///   not for events like global account data events or presence events
     ///
+    /// Returns an [`EventHandlerHandle`].
```

Comment:
This doesn't add anything, it should either be removed or extended with something like "that can be used to remove the event handler again".

Reply:
fair point. I think the documentation on `EventHandlerHandle` already states the latter, so I will remove this sentence entirely.